test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -869,23 +869,38 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
     expect(texts).not.toContain('<unknown>\n<anonymous>')
     expect(texts).toContain('foo\nbar (1:1)')
 
-    // Test that node:internal errors should be hidden
+    await cleanup()
+  })
 
-    next.patchFile(
-      'app/page.js',
-      // Node.js will throw an error about the invalid URL since this is a server component
-      outdent`
-      export default function Page() {
-        new URL("/", "invalid");
-      }`
+  test('should hide unrelated frames in stack trace with node:internal calls', async () => {
+    const { session, browser, cleanup } = await sandbox(
+      next,
+      new Map([
+        [
+          'app/page.js',
+          // Node.js will throw an error about the invalid URL since this is a server component
+          outdent`
+          export default function Page() {
+            new URL("/", "invalid");
+          }`,
+        ],
+      ])
     )
 
+    // Test that node:internal errors should be hidden
```

Comment:
```suggestion
    // Test that node:internal errors should be hidden
```

unnecessary comment since this is a separate test now